app/scripts/controllers/table-search-controller.js

```diff
@@ -211,6 +213,20 @@ angular.module('groongaAdminApp')
         }
       }
 
+      function toggleAllColumns() {
+        $scope.table.allColumns.forEach(function(columnInfo) {
+          columnInfo.output = $scope.toggleAllColumnsChecked;
+        });
+        if (!$scope.toggleAllColumnsChecked) {
+          $scope.table.allColumns.filter(function(columnInfo) {
```

Comment:
I'll check _id, _key and additional some columns in most cases, so I treat these columns as special cases.